post-install.yml

```diff
@@ -102,6 +102,16 @@
         git push {{ dashboard_git_remote.stdout }} master
       args: 
         chdir: /tmp/tsuru-dashboard
+      register: dashboard_push_result
+
+    - name: Tsuru set environment variables for dashboard
+      shell: >
+        tsuru env-set -a dashboard STATSD_PORT=8125;
+        tsuru env-set -a dashboard STATSD_HOST={{ deploy_env }}-metrics.{{ domain_name }};
+        tsuru env-set -a dashboard GRAPHITE_HOST={{ deploy_env }}-metrics.{{ domain_name }};
+        tsuru env-set -a dashboard GRAPHITE_USER={{ graphite_auth_user }}
+        tsuru env-set -a dashboard GRAPHITE_PASS={{ graphite_auth_pass }}
+      when: "not 'Everything up-to-date' in dashboard_push_result.stderr"
```

Comment:
Yes, this in general needs proper fix. The variables need to be set every time they are changed. This is unrelated to the app deploys, which is what is now checked. Proposed way: output them to file and on a file change, actually run that file which will in turn set the variables...